mockups/base.html

```diff
@@ -1,36 +0,0 @@
-<!DOCTYPE html>
```

Comment:
I was trying to delete stuff as I port it over, but I agree I should just leave the mockups until I'm completely done. :sparkles: